package.json

```diff
@@ -23,7 +23,8 @@
     "test.browser": "jest --runInBand --testPathPattern '__tests__/browser'",
     "watch": "webpack -w --progress --mode production",
     "release": "npx semantic-release",
-    "release.dry": "npx semantic-release --dry-run"
+    "release.dry": "npx semantic-release --dry-run",
+    "sync": "rdme docs ./docs --version=2"
```

Comment:
I didn't realize we made it so simple. That's cool.

Reply:
Yeah, our [CLI tools](https://github.com/readmeio/rdme#usage) are dope af.